sebs/utils.py

```
@@ -33,14 +72,36 @@ def default(self, o):
+    Serialize an object to a JSON string.
+    
+    Args:
+        obj: Object to serialize
+        
+    Returns:
+        str: JSON string representation of the object
+    """
     if hasattr(obj, "serialize"):
         return json.dumps(obj.serialize(), sort_keys=True, indent=2)
     else:
         return json.dumps(obj, cls=JSONSerializer, sort_keys=True, indent=2)
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)